src/api/activiti-rest-api/api/appDefinitions.api.ts

```diff
@@ -128,12 +128,14 @@ export class AppDefinitionsApi extends BaseApi {
      * @param file file
      * @return Promise<AppDefinitionUpdateResultRepresentation>
      */
-    importAndPublishApp(file: any): Promise<AppDefinitionUpdateResultRepresentation> {
+    importAndPublishApp(file: any, opts?: any): Promise<AppDefinitionUpdateResultRepresentation> {
         throwIfNotDefined(file, 'file');
 
         let postBody = null;
         let pathParams = {};
-        let queryParams = {};
+        let queryParams = {
+            'renewIdmEntries': opts['renewIdmEntries']
```

Comment:
the opts is marked as `any`, so accessing indexer will crash if it's not provided, try the following:
```suggestion
            'renewIdmEntries': opts?.['renewIdmEntries']
```

Reply:
thanks!